### ISSUE TITLE
Add lime3DS core

### ISSUE THREAD

**Issue by Algiuxs** [libretro/RetroArch#17094]
Add lime3ds core, it's vulkan and opengl renderer.

**Comment by Twhylite33** [comment]
I second this! On the citra emulator from what i can tell, you cannot change the resolution where on lime 3ds you can, making games much more enjoyable

**Comment by Algiuxs** [comment]
also lime3ds was renamed to azahar

**Comment by Leusele** [comment]
Yes, please try to add the Azahar core for Retroarch so we can use it on switch/android

I have no clue how much work that is but would love to see that <3

**Comment by persechini** [comment]
Azahar core would be amazing, we are missing a compelling 3ds core

**Comment by Algiuxs** [comment]
Some progress is happening at https://github.com/azahar-emu/azahar/pull/1215

**Comment by davidhedlund** [comment]
Reasons why Azahar is important:
* Currently Citra is still used as libretro core. But Citra is discontinued so eventually some fork of it has to replace it.
* [Azahar](https://github.com/azahar-emu/azahar) is the most popular Citra fork. The [Ahazhar GitHub repository](https://github.com/azahar-emu/azahar) has 5.5 k stars at this writing. Azahar is distributed under [GNU GPL version 2](https://github.com/azahar-emu/azahar).
* Azahar is the only Nintendo 3DS emulator that is distributed at [flathub](https://flathub.org/en/apps/org.azahar_emu.Azahar).

**Comment by Emulayr** [comment]
Made test build, and a release for Android to test:
https://github.com/Emulayr/azahar-libretro-builds
Works perfectly, performance is better than Citra

**Comment by gouchi** [comment]
@Algiuxs Please close this issue as the PR  has been merged in upstream project.

It will be available soon in the Online updater.

**Comment by davidhedlund** [comment]
> It will be available soon in the Online updater.

@gouchi About 47 days have passed, but it still hasn’t appeared. Is there something preventing it from being added, or can we expect it to show up later this year?

**Comment by warmenhoven** [comment]
It's there already.

**Comment by davidhedlund** [comment]
> It's there already.

Sorry, I see it now. I think I confused it with [Citron](https://github.com/libretro/RetroArch/issues/18477) again.

**Comment by davidhedlund** [comment]
@warmenhoven Now when Azahar is available, should these cores be removed from Core Downloader?:
* Citra (2018)
* Citra

**Comment by warmenhoven** [comment]
No, I think they have a place and a purpose, even if it's just so people can continue to use their saves and states. I think citra2018 also might support lower end devices? Also citra supports encrypted 3DS files.

**Comment by davidhedlund** [comment]
> No, I think they have a place and a purpose, even if it's just so people can continue to use their saves and states. I think citra2018 also might support lower end devices? Also citra supports encrypted 3DS files.

I just discovered a bug in Azahar that I filed here:
* https://github.com/libretro/RetroArch/issues/18911

**Comment by gabri324** [comment]
Disculpen será que saldrá core para retroarch en la switch? Sería muy bueno la verdad
